app/rtc_compare.py

```diff
@@ -262,14 +276,21 @@ def compare_hdf5_elements(hdf5_obj_1, hdf5_obj_2, str_key, is_attr=False,
     shape_val_1 = val_1.shape
     shape_val_2 = val_2.shape
 
+    # Start the comparison
+    if str_key in list_exclude:
```

Comment:
```suggestion
    if list_exclude is not None and str_key in list_exclude:
```
If you set the default of `list_exclude` to `None`, you may want to check if it's not `None` before using it.